package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "eslint-plugin-naming-convention",
+  "version": "1.1.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
```

Comment:
this should call `neutrino test` command

Reply:
Neutrino test command was added